index.html

```diff
@@ -92,25 +92,16 @@ <h3 class="projectName">Project One</h3></a>
 				</div>
 			</div>
 		</section>
-		<!-- Possible extra section>
-		<section class="featureProjects">
-			<div class="featureProjectsItem">
-			</div>
-			<div class="featureProjectsItem">
-			</div>
-			<div class="featureProjectsItem">
-			</div>
-			<div class="featureProjectsItem">
-			</div>
-		</section>
-			-->
 		<a class="anchor" id="contact"></a>
 		<section class="contactMe">
 			<div class="contactText">
-				<div class="contactTitle">
-				</div>
+				<h2 class="contactTitle">Contact me</h2>
+				<p class="contactDesc">If you'd like to get in touch, feel free to drop me an email. If you'd like to keep an eye out for future projects, take a look at my social channels.</p>
 			</div>
-			<div class="socialContainer">
+			<div class="contactBtnContain">
+				<a href="mailto:jordanaddis@gmail.com"><i class="contactBtn far fa-envelope"></i>
+				<a href="https://github.com/j-add" target="_blank"><i class="contactBtn fab fa-github socialIcon"></i></a>
+				<a href="http://linkedin.com" target="_blank"><i class="contactBtn fab fa-linkedin-in socialIcon"></i></a>
```

Comment:
is this supposed to point to your profile on linkedin?

Reply:
It is. I'll add it in and change to a https link.